cmd/completion/main.go

```diff
@@ -57,56 +63,112 @@ func init() {
 func main() {
 	flag.Parse()
 
-	if hasCosign() || notaryV1URL != "" {
-		err := signImage()
-		if err != nil {
-			log.Fatal(err)
-		}
+	var report platform.ExportReport
+	_, err := toml.DecodeFile(reportFilePath, &report)
+	if err != nil {
+		log.Fatal(err, "toml decode")
 	}
 
-	logger.Println("Build successful")
-}
+	if len(report.Image.Tags) == 0 {
+		log.Fatal(errors.New("no image found in report"))
+	}
 
-func signImage() error {
-	var report platform.ExportReport
-	_, err := toml.DecodeFile(reportFilePath, &report)
+	builtImageRef := fmt.Sprintf("%s@%s", report.Image.Tags[0], report.Image.Digest)
+
+	k8sNodeKeychain, err := k8schain.NewNoClient(context.Background())
 	if err != nil {
-		return errors.Wrap(err, "toml decode")
+		log.Fatal(err)
 	}
 
-	creds, err := dockercreds.ParseMountedAnnotatedSecrets(registrySecretsDir, dockerCredentials)
+	creds, err := dockercreds.ParseBasicAuthSecrets(registrySecretsDir, dockerCredentials)
```

Comment:
It is unfortunate we still need to do this vs just using the same <home>/.docker/config.json that the lifecycle uses.